spec/requests/graphql/mutations/batch_export/delete_batch_export_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::BatchExport::DeleteBatchExport, type: :request do
+  let(:user) { FactoryBot.create(:user) }
+  let(:file_location) { Hyacinth::Config.batch_export_storage.generate_new_location_uri('test-123') }
+  let(:batch_export) do
+    exp_job = FactoryBot.create(:batch_export, :success, user: user, file_location: file_location)
+    Hyacinth::Config.batch_export_storage.write(exp_job.file_location, 'Some content')
+    exp_job
+  end
+  let(:id) { batch_export.id }
+
+  include_examples 'requires user to have correct permissions for graphql request' do
+    let(:variables) { { input: { id: id } } }
+    let(:request) { graphql query, variables }
+  end
+
+  context 'when user is logged in' do
+    before do
+      login_as user, scope: :user
+      batch_export # create the batch export
```

Comment:
I don't think you need this because referencing `id` will generate the `batch_export`

Reply:
Ah, right!  Will remove, thanks!